lib/src/components/NxTag/NxTag.scss

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+ @import '../../scss-shared/nx-variables';
+ @import '../../scss-shared/nx-text-helpers';
+ @import '../../scss-shared/nx-container-helpers';
+
+.nx-tag {
+  @include semi-bold();
+  align-items: center;
+  background-color: $nx-blue-800;
+  border: 1px solid $nx-blue-800;
+  border-radius: 16px;
+  box-sizing: border-box;
+  color: $nx-text-color-dark;
+  cursor: default;
+  display: inline-block;
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-xs $nx-spacing-xs 0;
+  padding: 5px $nx-spacing-xs 6px $nx-spacing-md;
+  max-width: 824px;
```

Comment:
Remind me, how did we arrive at this number?

Reply:
> Remind me, how did we arrive at this number?

Apparently it is 58 capital W's?

@dturner32 ?

Reply:
> Apparently it is 58 capital W's?

But how did we arrive that _that_ being the metric we care about.  I feel like it was probably based on the length of some field in IQ that happens to show up as a tag in the UI.  But I don't know that we should be using that as our basis as that field is just one potential use case for tags.

Reply:
> > Apparently it is 58 capital W's?
> 
> But how did we arrive at _that_ being the metric we care about? I feel like it was probably based on the length of some field in IQ that happens to show up as a tag in the UI. But I don't know that we should be using that as our basis as that field is just one potential use case for tags.

Beats me, it's in Zeplin. @dturner32 ?

Reply:
that number was originally based on an example coming out of IQ. I remember Caleb mentioning something at the time around the max character width be 58. Still, I would want to cap the tag length (if we need to) so that we don't end up with super long tags. I'm thinking a round number like 900px as the max width

Reply:
> that number was originally based on an example coming out of IQ. I remember Caleb mentioning something at the time around the max character width be 58. Still, I would want to cap the tag length (if we need to) so that we don't end up with super long tags. I'm thinking a round number like 900px as the max width

To me the idea of a `900px` wide tag seems insane. Since we have implemented truncation and automatic tooltips when the text is too long I'd be far happier to come up with a `max-width` that much more appropriate to the concept of a tag

Reply:
I'm fine with that, the attached screenshot shows the tag width at 312px which is the width of it within the current sidebar. Taking truncation and and tooltips into consider the tag doesn't need to be 58 characters long. I suggest we round that 312 up and cap the tag width at 320px. Thoughts? 

![Screen Shot 2021-01-25 at 11 36 24 AM](https://user-images.githubusercontent.com/62309246/105735505-c59def00-5f01-11eb-9ecb-63fcec661e4c.png)

Reply:
> I'm fine with that, the attached screenshot shows the tag width at 312px which is the width of it within the current sidebar. Taking truncation and and tooltips into consider the tag doesn't need to be 58 characters long. I suggest we round that 312 up and cap the tag width at 320px. Thoughts?

The way I see it the options are either pick an arbitrary number or say it can go full width across its container.  If we want an arbitrary number 320 seems fine to me.  Of course, even with an arbitrary number we would also want it to be constrained by the container if the container has less room than the chosen arbitrary number.

Reply:
> Of course, even with an arbitrary number we would also want it to be constrained by the container if the container has less room than the chosen arbitrary number.

And we'd probably want and example proving that that works

Reply:
> > Of course, even with an arbitrary number we would also want it to be constrained by the container if the container has less room than the chosen arbitrary number.
> 
> And we'd probably want an example proving that that works

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/50c5188ab0d555395713e1994ff1c0008692d528